src/Postnl.php

```diff
@@ -329,6 +337,42 @@ public function currentStatus($barcode)
     }
 
     /**
+     * @param string $postalCode
+     * @param ComplexTypes\ArrayOfCutOffTime $cutOffTimes
+     * @param string $shippingDate
+     * @param int $shippingDuration
+     * @param string[] $options
+     * @param string $allowSundaySorting
+     * @param string $countryCode
+     */
+    public function getDeliveryDate(
+        $postalCode,
+        $cutOffTimes,
+        $shippingDate,
+        $shippingDuration = 1,
+        $options = ['Daytime'],
+        $allowSundaySorting = 'false',
+        $countryCode = 'NL'
+    ) {
+        /** @var ComplexTypes\GetDeliveryDate $GetDeliveryDate */
+        $GetDeliveryDate = ComplexTypes\GetDeliveryDate::create();
+        $GetDeliveryDate
```

Comment:
You can skip this and just do this instead (as the `setFoo` methods return `$this`):

```php
$GetDeliveryDate = ComplexTypes\GetDeliveryDate::create()
    ->setPostalCode($postalCode)
    .....
```